android/src/main/java/expo/modules/geofencingbatchplugin/GeofencingEventsReceiver.kt

```diff
@@ -0,0 +1,96 @@
+package expo.modules.geofencingbatchplugin
+
+import android.content.BroadcastReceiver
+import android.content.Context
+import android.content.Intent
+import android.util.Log
+import com.batch.android.Batch
+import com.batch.android.BatchEventAttributes
+import com.webgeoservices.woosmapgeofencingcore.database.POI
+import com.webgeoservices.woosmapgeofencingcore.database.WoosmapDb
+import org.json.JSONException
+import org.json.JSONObject
+import java.util.concurrent.ExecutorService
+import java.util.concurrent.Executors
+
+class GeofencingEventsReceiver : BroadcastReceiver() {
+
+    companion object {
+        private const val TAG = "GeofencingReceiver"
+    }
+
+    private val executorService: ExecutorService = Executors.newSingleThreadExecutor()
+
+    override fun onReceive(context: Context, intent: Intent) {
+        Log.d(TAG, "Received broadcast")
+        executorService.execute {
+            try {
+                Log.d(TAG, "executorService.execute")
```

Comment:
Data is different from what we specify in the woosmap documentation
https://developers.woosmap.com/products/geofencing-sdk/integration/batch/